### DISCUSSION TITLE
Private Rebuttals

### DISCUSSION THREAD

**Discussion by seporterfield** [Monstarrrr/rebutify#86] [General]
What are private rebuttals?
How do we want them to work?

Copy-pasted from discord (@Monstarrrr):

> Rebuttals you make but only you can see them.  
Draft rebuttals - if we made such a thing - would be private necessarily, but I was thinking about having rebuttals you post being marked as private instead of as draft.  

> The usage would be the same as draft, except unlike with the term ''draft'' we're not implying that it's unfinished or that the user has to eventually post it publicly.  
~~It also removes that potential ambiguity of ''if it's a draft then can I really trust the website to keep it indefinitely on the page or to allow me to save/organize it somewhere nicely or should I just post it publicly just to be sure?~~''  
Which would be a dark-pattern imo.

**Comment by seporterfield** [comment]
This affects the data structure introduced in #85

**Comment by seporterfield** [comment]
@Monstarrrr could you expand on this?

> ambiguity of ''if it's a draft then I don't trust the website to keep it indefinitely nor allow me to save/organize it somewhere nicely'

**Reply by Monstarrrr** [reply]
Thought about it more, this is a bias on my end from experience with certain services.  
I understand that in most popular apps "drafts" can be saved indefinitely and organized properly in a similar way as actual posts.

**Comment by seporterfield** [comment]
Clarification:
Private = actually post the rebuttal, but no one else can see/interact with it (this one)

Draft = saved but not posted (not this one)

**Comment by Monstarrrr** [comment]
Had time to think more about the case for having ''draft'' **instead of** ''private rebuttal'' like you suggested @seporterfield and I don't see a case in which using ''draft'' in the back-end causes inconveniences
